### PR TITLE
No more mapnik

### PR DESCRIPTION
**Pull request by chelm**
This PR removes the mapnik dependency which was a big install and blocked many windows users. Koop still supports tiles but will do via a new koop-tile-plugin. 

The idea of plugins are to provide extra functionality on top of the core node features. Means we now have a pattern to do some wild and crazy things w/o impacting the koop core module.

### CODE REVIEW

**Comment by chelm** [comment]
Builds are stilling failing...

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0] - 2015-03-17
+### Changed
+* No more mapnik dependency by default. Instead all tile logic has been moved to [koop-tiles-plugin](https://github.com/koopjs/koop-tile-provider)
```
Comment (ngoldman) [inline CHANGELOG.md]: Couple typos: you named it koop-tile-plugin and it's at https://github.com/koopjs/koop-tile-plugin

Comment (chelm) [inline CHANGELOG.md]: oh nice

**Comment by chelm** [comment]
@ngoldman Got the tests passing. I still need to revisit writing to s3 via travis. will do that tonight. Merge?

**Comment by ngoldman** [comment]
:+1:

**Comment by ngoldman** [comment]
@chelm Just realized if tiles are no longer in koop by default and users are required to use `koop-tile-plugin` on there own, this is a breaking change.

**Comment by chelm** [comment]
eww you are right. Users will have to include the plugin so we need to up the version. 

@ngoldman what's the best way to resolve that? Can we go back and tag it as a major version update?

**Comment by ngoldman** [comment]
@chelm once something's been released -- *there's no going back*

generally in nodeland it's encouraged to keep moving forward rather than changing history. we could..

1. add a warning to the changelog and act like everything's cool
2. bump a major version and add a deprecation warning for 1.1.x
3. add koop-tile-plugin back as a dependency to 1.2.x to avoid forcing a major update and warn/deprecate for 1.1.x

**Pull request merged**